### PR TITLE
Use tsconfig path mappings

### PR DESCRIPTION
**Pull request by qwertyuu**
#### Changes

- A typo in the readme
- Added tsconfig path mappings to get rid of the infamous "../../../" hell
- Converted all paths to use them
    - Only exception is the `svg` imports as typescript does not compile them

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (not needed)
- [x] Tests (not needed)

### CODE REVIEW

**Review by GabLeRoux — APPROVED** [review]
<3

**Comment by GabLeRoux** [comment]
I'm not sure why, but vercel failed to deploy this pr, that's strange.

**Comment by qwertyuu** [comment]
@GabLeRoux yep and I can't get any info on why whatsoever

**Comment by qwertyuu** [comment]
Can you rerun the action manually, as with a CI?

**Comment by qwertyuu** [comment]
@GabLeRoux I think I found the root of the problem. I tried to run `yarn build` and it had errors. Now it is fine, let's see again

**Comment by qwertyuu** [comment]
There you go

**Review by GabLeRoux — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -12,7 +13,12 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "paths": {
+      "@/core/*": ["core/*"],
+      "@/layout/*": ["components/layout/*"],
+      "@/markdown/*": ["components/markdown/*"],
```
Comment (webbertakken) [inline tsconfig.json:20]: These last 2 may be a bit too specific and become out of date every time we add a component to the website.

I would suggest changing it to @/components instead.

**Review by webbertakken — CHANGES_REQUESTED** [review]

**Comment by webbertakken** [comment]
LGTM apart from the one comment! Thank you @qwertyuu

File: tsconfig.json (hunk 1 above)
Comment (qwertyuu) [reply tsconfig.json:20]: Aight, I'll do that soon :D

**Review by webbertakken — APPROVED** [review]
Great, thank you @qwertyuu

**Pull request merged**